### ISSUE TITLE
pysatMagVect tests require pysat, but pysat is not a dependancy in setup.py

### ISSUE THREAD

**Issue by lkilcommons** [rstoneback/pysatMagVect#13]
After a successful installation `python setup.py install`,
`nosetest -vs pysatMagVect` fails with `No module pysat`. Minor problem, but might be good if pysat were added to optional dependancies in setup.py.
e.g.
```
'tests_require=['pysat']'
```
or
```
extras_require={'testing':['nose','pysat']}
```
Not clear which is best practice, I've seen both. I think tests_require is specifically if you want `python setup.py test` to run your tests. https://stackoverflow.com/questions/26064738/should-pytest-et-al-go-in-tests-require-or-extras-requiretesting 

(Python 2.7 in virtual environment on Glade supercomputer)

**Comment by rstoneback** [comment]
Thanks for the report Liam.

I’ll take a look. Just a heads up that develop is the most current and has a number of changes compared to master. I’m planning on pulling into master very soon, once I get back to Dallas and have some time to catch up.

Cheers,
Russell

On Nov 16, 2018, at 11:08 AM, Liam Kilcommons <notifications@github.com<mailto:notifications@github.com>> wrote:


After a successful installation python setup.py install,
nosetest -vs pysatMagVect fails with No module pysat. Minor problem, but might be good if pysat were added to optional dependancies in setup.py.
e.g.

'tests_require=[pysat]'


or

extras_require={'testing':[nose,pysat]}


Not clear which is best practice, I've seen both. I think tests_require is specifically if you want python setup.py test to run your tests. https://stackoverflow.com/questions/26064738/should-pytest-et-al-go-in-tests-require-or-extras-requiretesting

**Comment by rstoneback** [comment]
Fix in pull #14 . I added pysat to the general requirements and nose via the extras_require. Thanks for posting for the issue.
